docs/SetupDatabaseForAzure.md

```diff
@@ -48,3 +48,24 @@ You can choose Flexible or Single server deployment for Azure Database for MySQL
 Note:-
 
 * The [Storage auto-grow](https://docs.microsoft.com/en-gb/azure/mysql/concepts-pricing-tiers#storage-auto-grow) feature automatically increases storage without impacting the workload and is enabled by default in server configuration while creating an Azure Database for MySQL instance.
+
+
+## Azure Database for PostgreSQL
+
+In this section, you will create an Azure Database for PostgreSQL.
+
+### Steps
+
+You can choose Flexible or Single server deployment for Azure Database for PostgreSQL, follow the [Azure official guide](https://docs.microsoft.com/en-us/azure/postgresql/overview-postgres-choose-server-options) to learn more.
```

Comment:
```suggestion
You need to choose Flexible or Single server deployment for Azure Database for PostgreSQL. Read the [Azure official guide](https://docs.microsoft.com/en-us/azure/postgresql/overview-postgres-choose-server-options) to learn more.
```
`You can choose` means, you can choose it or you don't neet choose it, so it's misleading.
If users need to choose one or the other, you should say, `you need to`.
I've said this several times.

Reply:
Sorry for the mistake.
I have updated the sentence.